msgraphcore/middleware/authorization.py

```diff
@@ -29,25 +29,9 @@ def _get_middleware_options(self, request):
 
 
 class TokenCredentialAuthProvider(AuthProviderBase):
-    def __init__(self, credential: TokenCredential, scopes: [str] = BASE_URL + '.default'):
+    def __init__(self, credential: TokenCredential, scopes: [str] = ['user.read']):
```

Comment:
Why did you make this change from default to user.read?

Reply:
I just realized that we need to also consider incremental consent. During the life time of the application, as the application gets updated, or a user wants to access new functionality, the app could need to support incremental user consent. Do we have access to set the scope on the auth handler after it has been added to the middleware?

Reply:
Yes, the design supports incremental consent.